packages/passwordless-auth-construct/src/services/ses_service.ts

```diff
@@ -0,0 +1,72 @@
+import { SESClient, SendEmailCommand } from '@aws-sdk/client-ses';
+import { logger } from '../logger.js';
+import { DeliveryMedium, DeliveryService, SesServiceConfig } from '../types.js';
+
+/**
+ * SNS service Implementation.
+ * Sends message via SNS client.
+ */
+export class SesService implements DeliveryService {
+  deliveryMedium: DeliveryMedium = 'EMAIL';
+
+  /**
+   * SES Service constructor
+   * @param sesClient - SES Client to use for sending messages. Defaults to SESClient. Can be overridden for testing.
+   * @param config - SES Service configuration
+   */
+  constructor(private sesClient: SESClient, private config: SesServiceConfig) {}
+
+  public send = async (message: string, destination: string): Promise<void> => {
+    // TODO: Add support for custom email input
+    const emailCommand = new SendEmailCommand({
+      Destination: { ToAddresses: [destination] },
+      Message: {
+        Body: {
+          Html: {
+            Charset: 'UTF-8',
+            Data: message,
+          },
+          Text: {
+            Charset: 'UTF-8',
+            Data: message,
+          },
+        },
+        Subject: {
+          Charset: 'UTF-8',
+          Data: this.config.emailSubject,
+        },
+      },
+      Source: this.config.fromAddress,
+    });
+
+    // Send Email via SES
+    const output = await this.sesClient.send(emailCommand);
+    logger.debug(`SMS sent: ${JSON.stringify(output, null, 2)}`);
+  };
+
+  public mask = (destination: string): string => {
```

Comment:
*note[non-blocking]:* I am noticing that the logic here is not quite the same as Cognito. I don't think that is necessarily an issue but ideally they would be the same. I am going to create a task to investigate and align them if it makes sense.